assets/dev/scss/frontend/widgets/google-maps.scss

```diff
@@ -1,6 +1,15 @@
+//
+// Google Map
+//
+
 .elementor-widget-google_maps {
 
+	.elementor-widget-container {
+		overflow: hidden; // Allow border-radius
+		transform: translate3d(0, 0, 0); // Allow border-radius in Safari
```

Comment:
Remove it as discussed

Reply:
Fixed: https://github.com/elementor/elementor/pull/12199/commits/62890eb309d4255ae8609628d6cd95c11f515487